hasher-matcher-actioner/Makefile

```diff
@@ -11,17 +11,28 @@ REPOSITORY_NAME = hma-lambda-dev
 DOCKER_TAG ?= ${USER} # todo - change to ${USER}-latest?
 DOCKER_URI = $(call shell-or-die,docker images --filter=reference='*/${REPOSITORY_NAME}:${DOCKER_TAG}' --format='{{.Repository}}')
 
+LOCAL_TE_DIR = local_threatexchange # Should match LOCAL_THREAT_EXCHANGE ARG in Dockerfile
 
 all:
 	@echo >&2 "Must specify target. ${FAIL}"
 
 docker:
+	if [ ! -d ${LOCAL_TE_DIR} ] ; then mkdir ${LOCAL_TE_DIR} ; fi
```

Comment:
Hmm, not a fan of this. Would rather have two dockerfiles. Does Dockerfile.dev solve this issue? https://stackoverflow.com/questions/39093806/what-is-a-dockerfile-dev-and-how-is-it-different-from-dockerfile